functions/slack/index.js

```diff
@@ -106,26 +106,23 @@ const verifyWebhook = body => {
  *
  * @param {string} query The user's search query.
  */
-const makeSearchRequest = query => {
-  return new Promise((resolve, reject) => {
-    kgsearch.entities.search(
-      {
-        auth: config.KG_API_KEY,
-        query: query,
-        limit: 1,
-      },
-      (err, response) => {
-        console.log(err);
-        if (err) {
-          reject(err);
-          return;
-        }
-
-        // Return a formatted message
-        resolve(formatSlackMessage(query, response));
+const makeSearchRequest = async query => {
+  await kgsearch.entities.search(
```

Comment:
1) Do we know if this _actually_ returns a promise? (either via running it ourselves, or reviewing Slack's docs)
2) If so, we should get rid of the callback below. 🙂